.builds/freebsd.yml

```diff
@@ -0,0 +1,45 @@
+image: freebsd/latest
+sources:
+- https://github.com/mesonbuild/meson
+environment:
+  CI: 1
+  MESON_CI_JOBNAME: thirdparty
+packages:
+- python
+- python
+- py39-setuptools
+- py39-pytest-xdist
+- ninja
+- pkgconf
+- cmake
+- gtk3
+# languages
+- vala
+- rust
+- mono
+- py39-cython
+# frameworks
+- sdl2
+- gpgme
+- gettext
+- ncurses
+- boost-all
+- qt5-buildtools
+- gobject-introspection
+tasks:
+- command-tests: |
+    cd meson
+    ./run_meson_command_tests.py -v || touch ~/.failed.marker
+- unittests: |
+    cd meson
+    ./run_unittests.py -ra || touch ~/.failed.marker
+- unittests-root: |
+    cd meson
+    sudo CI=$CI MESON_CI_JOBNAME=$MESON_CI_JOBNAME ./run_unittests.py -ra || touch ~/.failed.marker
+- project-tests: |
+    cd meson
+    ./run_project_tests.py || touch ~/.failed.marker
+- project-tests-root: |
+    cd meson
+    sudo CI=$CI MESON_CI_JOBNAME=$MESON_CI_JOBNAME ./run_project_tests.py --only common
+    ! test -f ~/.failed.marker
```

Comment:
nitpick: seems a tad odd to use a different syntax here than a few lines above

Reply:
It's not really different syntax. Above, it's "do this action, or do that other action". Here, it's "do this action, and only this action, but reverse its status".

As a python analogy, above I do `with return_1_is_error(): return run_tests() or mark_failure()`, while here I do `with return_1_is_error(): return not fail_file_exists()`.

I *could* rewrite it as `with return_1_is_error(): return fail_file_exists() or return_1()`, in theory.